site/content/docs/api-checks/setup-script-examples.md

```diff
@@ -261,25 +256,26 @@ request.headers['Authorization'] = `Bearer ${access_token}`
 {{< /tab >}}
 {{< tab "JavaScript" >}}
 ```js
-const axios = require('axios').default
+const axios = require('axios')
 
 // grab the necessary credentials set up earlier in your environment variables.
-const { ISSUER, USERNAME, PASSWORD, CLIENT_ID, CLIENT_SECRET, AUDIENCE } = process.env
-
+const { AUDIENCE, CLIENT_SECRET, CLIENT_ID, ISSUER, USERNAME, PASSWORD, SCOPE } = process.env;
 
 // fetch an access token
-const { data: { access_token }} = await axios({
-  url: ISSUER,
-  method: 'POST',
-  data: {
-    grant_type: 'password',
-    username: CHECKLY_USER,
-    password: CHECKLY_PWD,
-    client_id: CLIENT_ID,
-    client_secret: CLIENT_SECRET,
-    audience: AUDIENCE
-  },
-})
+const { data: { access_token } } = await axios.post(`${ISSUER}/oauth/token`,
+    {
+      grant_type: 'password',
+      username: `${USERNAME}`,
+      password: `${PASSWORD}`,
+      audience: `${AUDIENCE}`,
+      scope: `${SCOPE}`,
+      client_id: `${CLIENT_ID}`,
+      client_secret: `${CLIENT_SECRET}`
```

Comment:
Are client and secret still needed in this password auth flow?

Reply:
auth0 password grant docs - [Example POST to token URL](https://auth0.com/docs/get-started/authentication-and-authorization-flow/resource-owner-password-flow/call-your-api-using-resource-owner-password-flow#example-post-to-token-url)

I also attempted to run the test script from my CLI with `client_id` and/or `client_script` omitted and ran into a `401` error. Check only passes with _**both**_ properties included.